### PR TITLE
[Enhancement] Add loading error to error_url

### PR DESCRIPTION
**Pull request by rickif**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

This PR adds all load error to the error_url, to make it easier to inspect the loading problem.

### CODE REVIEW

File: be/src/runtime/stream_load/stream_load_executor.cpp [hunk 1]
```diff
@@ -111,6 +106,10 @@ Status StreamLoadExecutor::execute_plan_fragment(StreamLoadContext* ctx) {
                 ctx->write_data_cost_nanos = MonotonicNanos() - ctx->start_write_data_nanos;
                 ctx->promise.set_value(status);
 
+                if (!executor->runtime_state()->get_error_log_file_path().empty()) {
```
Comment (imay) [inline be/src/runtime/stream_load/stream_load_executor.cpp:109]: why are you removing `ctx->number_filtered_rows > 0`?

Comment (rickif) [reply be/src/runtime/stream_load/stream_load_executor.cpp:109]: I notice that the result of `ctx->number_filtered_rows > 0` is not strictly related to `executor->runtime_state()->get_error_log_file_path()`.

Comment (imay) [reply be/src/runtime/stream_load/stream_load_executor.cpp:109]: IMO the error message is only useful when there is filtered rows. When the filtered rows is 0, no need to create this URL

**Review by imay — APPROVED** [review]

**Review by trueeyu — APPROVED** [review]

**Comment by rickif** [comment]
@mergyfio backport branch-2.3 branch-2.2

**Comment by rickif** [comment]
@mergifyio backport branch-2.3 branch-2.2

**Pull request merged**